cli/src/generate/templates/setup.py

```diff
@@ -0,0 +1,52 @@
+from os.path import join
+from setuptools import Extension, find_packages, setup
+from setuptools.command.build import build
+from wheel.bdist_wheel import bdist_wheel
+
+
+class Build(build):
+    def run(self):
+        dest = join(self.build_lib, "tree_sitter_PARSER_NAME", "queries")
+        self.copy_tree("queries", dest)
+        super().run()
+
+
+class BdistWheel(bdist_wheel):
+    def get_tag(self):
+        python, abi, platform = super().get_tag()
+        if python.startswith("cp"):
+            python, abi = "cp38", "abi3"
+        return python, abi, platform
+
+
+setup(
+    packages=find_packages("bindings/python"),
+    package_dir={"": "bindings/python"},
+    package_data={
+        "tree_sitter_LOWER_PARSER_NAME": ["*.pyi", "py.typed"],
+        "tree_sitter_LOWER_PARSER_NAME.queries": ["*.scm"],
+    },
+    ext_package="tree_sitter_LOWER_PARSER_NAME",
+    ext_modules=[
+      Extension(
+        name="_binding",
+        sources=[
+            "bindings/python/binding.c",
```

Comment:
I've tried to use this at commit a9a7b9a4689e5581d6a8743c9d94b6fbeb47cd25 over tree-sitter-bash, and it was failing for me to compile when building the wheel using `pip wheel .`

<details><summary>Output of command: pip wheel .</summary>
<p>

```
Processing /tmp/tree-sitter-bash
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: tree-sitter-bash
  Building wheel for tree-sitter-bash (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for tree-sitter-bash (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [21 lines of output]
      running bdist_wheel
      running build
      copying queries/highlights.scm -> build/lib.linux-x86_64-cpython-311/tree_sitter_bash/queries
      running build_py
      copying bindings/python/tree_sitter_bash/__init__.py -> build/lib.linux-x86_64-cpython-311/tree_sitter_bash
      running egg_info
      writing bindings/python/tree_sitter_bash.egg-info/PKG-INFO
      writing dependency_links to bindings/python/tree_sitter_bash.egg-info/dependency_links.txt
      writing top-level names to bindings/python/tree_sitter_bash.egg-info/top_level.txt
      reading manifest file 'bindings/python/tree_sitter_bash.egg-info/SOURCES.txt'
      adding license file 'LICENSE'
      writing manifest file 'bindings/python/tree_sitter_bash.egg-info/SOURCES.txt'
      copying bindings/python/tree_sitter_bash/__init__.pyi -> build/lib.linux-x86_64-cpython-311/tree_sitter_bash
      copying bindings/python/tree_sitter_bash/binding.c -> build/lib.linux-x86_64-cpython-311/tree_sitter_bash
      copying bindings/python/tree_sitter_bash/py.typed -> build/lib.linux-x86_64-cpython-311/tree_sitter_bash
      running build_ext
      building '_binding' extension
      x86_64-pc-linux-gnu-gcc -Wsign-compare -DNDEBUG -march=native -mtune=native -O2 -pipe -fPIC -DPy_LIMITED_API=0x03080000 -DPY_SSIZE_T_CLEAN -Isrc -I/tmp/venv/include -I/usr/include/python3.11 -c bindings/python/binding.c -o build/temp.linux-x86_64-cpython-311/bindings/python/binding.o -std=c11
      cc1: fatal error: bindings/python/binding.c: No such file or directory
      compilation terminated.
      error: command '/usr/bin/x86_64-pc-linux-gnu-gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for tree-sitter-bash
Failed to build tree-sitter-bash                                                                                                                                                                                   
ERROR: Failed to build one or more wheels
```


</p>
</details> 

When I looked for that file, I found it under `bindings/python/tree_sitter_bash/binding.c`, so I guess we are missing here the directory. Adding it made it compile.

I then didn't forget to add `scanner.c`, and now it fully works after the change above:

```
$ pip wheel .
$ pip install tree_sitter_bash-0.0.1-cp38-abi3-linux_x86_64.whl
$ python3.11
>>> import tree_sitter_bash
>>> tree_sitter_bash.language()
140029515652384
```